frontend/Makefile

```diff
@@ -50,9 +50,13 @@ smoke-test-with-local-backend-ci:
 e2e:
 	npm run e2e-$(DEPLOYMENT_STAGE)
 
+.PHONY: e2e-on-local-dev
+e2e-on-local-dev:
```

Comment:
Just a comment / point for discussion: We currently have make targets that are expected to be run sometimes inside containers and sometimes outside.  I think this has gotten a little confusing / assumes a lot of background for the engineers running `make` commands. I propose that we set up a convention where the Makefile in the root of the repo should expose targets that are meant to be run _outside_ of containers, and the Makefiles in the frontend/backend/other dirs should be expected to run _inside_ their respective containers.

Reply:
Ahh that's a great idea and makes total sense 🔥 Encapsulation FTW 🎉 !